tests/acceptance/standard/test_group_avatar.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pytest
+from tests.acceptance import (
+    run_gitlabform,
+)
+
+
+class TestGroupAvatar:
+    def setup_method(self):
+        # Use gitlabform logo
+        self.project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../.."))
+        self.test_image_path = os.path.join(self.project_root, "docs/images/gitlabform-logo.png")
+
+        # Check if the file exists
+        assert os.path.exists(self.test_image_path), f"Test image not found at {self.test_image_path}"
+
+    def test__group_avatar_set(self, group):
+        # Test setting a group avatar
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config, group)
+
+        # Refresh group data
+        group = group.manager.get(group.id)
+
+        # Verify avatar is set
+        assert group.avatar_url is not None
+
+    def test__group_avatar_delete(self, group):
+        # Refresh group data to get current state
+        group = group.manager.get(group.id)
+
+        # Verify that avatar exists (should be set by previous test)
+        assert group.avatar_url is not None, "Avatar should exist from previous test"
+
+        # Delete the avatar
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: ""
+        """
+        run_gitlabform(config, group)
+
+        # Refresh group data
+        group = group.manager.get(group.id)
+
+        # Verify avatar is removed
+        assert group.avatar_url is None or "gravatar" in group.avatar_url
+
+    def test__group_avatar_file_not_found(self, group):
+        # Test handling of non-existent avatar file path
+        nonexistent_path = "/path/to/nonexistent/image.png"
+
+        # Store the original avatar URL to verify it doesn't change
+        group_original = group.manager.get(group.id)
+        original_avatar_url = group_original.avatar_url
+
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{nonexistent_path}"
+        """
+
+        # This should run without crashing, even though the file doesn't exist
+        run_gitlabform(config, group)
```

Comment:
I haven't actually thought about this before. Do you recall if this was already discussed? Why do we not want to fail when invalid avatar file path is provided in the config?

GitLabForm can continue running for processing other configs, but exit with non-zero to indicate there were failures. Otherwise, this will be a silent failure, which is not good. I think this is supposed to be the default behaviour, but not sure where/how this is configured. To validate this, additional config is needed though for the test (e.g. group setting or a project setting and validate they have been processed).

Ideally this can also be caught by the dry-run/noop feature, but we can leave that for a separate PR, if you like.